ebean-core/src/main/java/io/ebeaninternal/server/query/DbOrderByTrim.java

```diff
@@ -4,7 +4,7 @@
 
 final class DbOrderByTrim {
 
-  private static final Pattern orderByTrim = Pattern.compile("(?i) asc\\b| desc\\b|\\b nulls first\\b|\\b nulls last\\b");
+  private static final Pattern orderByTrim = Pattern.compile("(?i) asc\\b| desc\\b| nulls first\\b| nulls last\\b");
```

Comment:
this will not remove `.order("min(customer) asc nulls last")` properly.

Reply:
CHECKME: 
See TestOrderByParse: What about `nulls high` & `nulls low`... is this valid SQL?
What about collations?